contracts/Aavegotchi/libraries/LibWhitelist.sol

```diff
@@ -65,4 +67,29 @@ library LibWhitelist {
             _removeAddressFromWhitelist(_whitelistId, _whitelistAddresses[i]);
         }
     }
+
+    function setWhitelistAccessRight(
+        uint32 _whitelistId,
+        uint256 _actionRight,
+        uint256 _accessRight
+    ) internal {
+        require(_isAccessRightValid(_actionRight, _accessRight), "LibWhitelist: Invalid Rights");
+        AppStorage storage s = LibAppStorage.diamondStorage();
+        s.whitelistAccessRights[_whitelistId][_actionRight] = _accessRight;
+        emit WhitelistAccessRightSet(_whitelistId, _actionRight, _accessRight);
+    }
+
+    function _isAccessRightValid(uint256 _actionRight, uint256 _accessRight) internal pure returns (bool) {
+        // This action right limits borrowers in a whitelist to a number of borrowed gotchis. 0 is unlimited
+        if (_actionRight == 0) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    function borrowLimit(uint32 _whitelistId) internal view returns (uint256) {

```

Comment:
@pakim249CAL Is this function needed?
Looks like used once.

Reply:
Not really needed, but I think it's clearer from a dev perspective as it makes clear to future maintainers how to obtain the borrow limit.